src/components/Wizard/NewEditImportPane.tsx

```diff
@@ -108,11 +120,69 @@ function NewForm({ onNewMessage }: NewFormProps) {
   );
 }
 
-function EditForm() {
+function EditForm({ onEditMessage, onDeleteMessage, messages }: EditFormProps) {
+  const formContext = useForm<EditFormData>({ mode: "onChange" });
+  const {
+    register,
+    handleSubmit,
+    formState: { isDirty, isValid },
+  } = formContext;
+
+  const onSubmit: SubmitHandler<EditFormData> = (data: EditFormData) => {
+    onEditMessage(data.id);
+  };
+
+  const onDelete = (id: string) => {
+    onDeleteMessage(id);
+  };
   return (
     <>
       <Card.Title>Edit an Existing Message</Card.Title>
-      <Card.Text>There are no messages saved.</Card.Text>
+
+      <Form onSubmit={handleSubmit(onSubmit)} className="edit-form">
+        <FormProvider {...formContext}>
+          <Form.Group controlId="message-select">
+            <FormRow label="Select Message">
```

Comment:
`containerClassName="form-input-check"`

This will fix the vertical alignment issue